.golangci.yml

```diff
@@ -123,6 +123,8 @@ issues:
     - linters:
         - gocritic
       text: "^hugeParam: req is heavy"
+    - path: "_schema\\.go" # exclude rules for schema files as it's auto-genereated from OpenAPI spec
```

Comment:
IMO: I don't like this change.
At least for now, I prefer doing these manual updates to the schema file and ensure linting passes.
Then later when we introduce re-regenerating in the CI we can change this to be more permissive

Reply:
they are too many changes to do, I've created a ticket to follow it, I hope that's fine: CLOUDP-278408